Riot/Modules/KeyVerification/Common/KeyVerificationKind.swift

```diff
@@ -16,6 +16,26 @@
 import Foundation
 
 enum KeyVerificationKind {
-    case device
-    case user
+    case otherSession // An other session
```

Comment:
There is also the case to verify a session from a user that has not enabled cross-signing.
I think we fallback to this case. The title "Verify the other session" may be then not great. "Verify session" looked better in this situation.

Reply:
Right, it displays:
![image](https://user-images.githubusercontent.com/8418515/80458159-a6514180-8930-11ea-9d8c-8caf7a89bd85.png)

Reply:
You're right "Verify session" is more succinct in context, for a title.